app/views/shared/_footer.html.erb

```diff
@@ -8,8 +8,8 @@
         <h2 class="text-sm text-gray-500 dark:text-gray-400">The Odin Project is funded by the community. Join us in empowering learners around the globe by supporting The Odin Project!</h2>
       </div>
       <div class="text-center mt-4">
```

Comment:
Thanks for doing this @Asartea, definitely agree on giving these more space.

It looks like the margin on the outer edges is causing a bit of a responsive bug by pushing the buttons to a new line:
<img width="379" alt="Screenshot 2023-08-09 at 09 32 47" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/38cb231d-7713-47bf-841c-9953d6d31aaf">


But to simplify - Tailwind has a handy `space-` utility we can put on the the parent and take care of the gap and save us from giving each child element a margin.
 
```suggestion
      <div class="mt-4 flex justify-center space-x-6">
```